docs/weaver_code_anatomy.md

```diff
@@ -5,82 +5,91 @@ sidebar_label: View the generated code
 mandatory: True
 code: |
   interview_metadata # make sure we initialize the object
   if not defined("interview_metadata['a_258e_motion_for_impoundment']"):
     interview_metadata.initializeObject('a_258e_motion_for_impoundment')
   interview_metadata['a_258e_motion_for_impoundment'].update({
-    'title': '209a 258e motion for impoundment',
-    'short title': '209a 258e motion for impoundment',
-    'description': '209a 258e motion for impoundment',
-    'original_form': '',
-    'allowed courts': [
-      'Land Court',
+    "al_weaver_version": "0.82",
+    "generated on": "2021-05-28",
+    "title": "209a 258e motion for impoundment",
+    "short title": "209a 258e motion for impoundment",
+    "description": "209a 258e motion for impoundment",
+    "original_form": "",
+    "allowed courts": [
+      "Land Court",
     ],
-    'categories': [
-      'Guardianship',
+    "categories": [
+      "BE-04-00-00-00",
     ],
-    'logic block variable': 'a_258e_motion_for_impoundment',
-    'attachment block variable': 'a_258e_motion_for_impoundment_attachment',
-    'typical role': 'plaintiff',
+    "logic block variable": "a_258e_motion_for_impoundment",
+    "attachment block variable": "a_258e_motion_for_impoundment_attachment",
+    "typical role": "plaintiff",
   })
 ---
 ```
 
+1. `title`, `short title`, and `description` allow your organization's site to show more information about your form and to organize your forms more easily.
+1. `original_form` is simply a link to the original, fillable PDF form that this interview is automating, if it exists.
+1. 🚧 `allowed courts` allows your code to decide which courts to let the user pick from when they need to pick their court, usually used in conjunction with [AL Generic Jurisdiction](https://github.com/SuffolkLITLab/docassemble-ALGenericJurisdiction).
+1. `categories` is the [LIST taxonomy](https://taxonomy.legal/) code for this interview, which can be used by your organization to organize your interviews.
+1. `attachment block variable` used to be used in the code that sends documents to courts, but now the [ALDocument object block](#aldocument-object-block) is used instead.
+1. `logic block variable` should also no longer be used.
+1. `typical role`: controls which questions the user gets asked about themselves and other parties. 
+
 ## Main intro page
-Adds this text to the organization's intro page that appears at the beginning of every interview. This lets your user know right away that they have gotten to the right form (or the wrong one).
+Adds this text to the organization's intro page that appears at the beginning of every interview. This lets your user know right away that they have gotten to the right (or wrong) form. Note that this can (and should) be a more direct and detailed call to action, e.g. ("File a ___" or "Ask the court for ____"), rather than a simple short title, like the short title in the [metadata block](#metadata).
```

Comment:
Formatting
```suggestion
Adds this text to the organization's intro page that appears at the beginning of every interview. This lets your user know right away that they have gotten to the right (or wrong) form. Note that this can (and should) be a more direct and detailed call to action, e.g. ("File a \_\_\_\_" or "Ask the court for \_\_\_\_"), rather than a simple short title, like the short title in the [metadata block](#metadata).
```